gr-digital/lib/symbol_sync_cc_impl.cc

```diff
@@ -105,14 +97,14 @@ symbol_sync_cc_impl::symbol_sync_cc_impl(enum ted_type detector_type,
         throw std::out_of_range("output samples per symbol must be > 0");
 
     // Timing Error Detector
-    d_ted = timing_error_detector::make(detector_type, slicer);
-    if (d_ted == NULL)
+    d_ted.reset(timing_error_detector::make(detector_type, slicer));
+    if (d_ted == nullptr)
```

Comment:
{}

Reply:
also, un-idiomatic: `d_ted` is a `unique_ptr`, and as such has `operator bool` that does exactly this, so just `if(!d_ted)` would be the right thing.